modules/virt-about-qemu-guest-agent-web.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * .adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual machine overview* tab displays information about the *Operating system*, *Time zone*, and *Logged in users*.
```

Comment:
The guest agent also provides the hostname of the VM.

Reply:
Included Hostname as one of the items displayed in the tab.